azure_functions_worker/main.py

```diff
@@ -36,6 +31,14 @@ def parse_args():
 
 
 def main():
+    from .utils.dependency import DependencyManager
+    DependencyManager.initialize()
+    DependencyManager.use_worker_dependencies()
+
+    from . import logging
+    from ._thirdparty import aio_compat
+    from .logging import error_logger, logger
```

Comment:
Since the .logging module is defined by our own, I'm afraid at some point the .logging module will also use google.protobuf. So, just want to make sure all packages we imported from azure_functions_worker are from worker dependency path.